source/_posts/2022-01-07-android-quest.markdown

```diff
@@ -0,0 +1,63 @@
+---
+title: "Oculus Quest Meet the Smart Home!"
+description: "Automate your home based on how you use your Quest!"
+date: 2022-01-07 00:00:00
+date_formatted: "January 07, 2022"
+comments: true
+author: Daniel Shokouhi
+categories: Release-Notes
+og_image: /images/blog/2022-01-android-quest/Companion.png
+---
+
+
+![Screenshot of the Android app](/images/blog/2022-01-android/Companion.png)
+
+Hey Everyone! We hope you all had a great new years and holidays spent with the family! This past holiday season had some pretty cool toys going around. Cool toys to us mean more things to tinker with. One of the most popular gifts this year was no doubt the Oculus Quest and to be honest, how could it not be right? Naturally, working on the Android app some of us wondered how does the app performs on such a device? Well, it actually works pretty well and you can start building automations based on your usage today!
+
+<a href="https://sidequestvr.com/app/6427/home-assistant" style="display:inline-block"><img width="200" class="download-badge" alt="Get it on SideQuest" src="https://sidequestvr.com/assets/images/branding/Get-it-on-SIDEQUEST.png" style='box-shadow:none;border:0'></a>
+
+
+## Hello Sidequest!
+
+Wait Sidequest? What is that? Why is this app not on the Oculus Quest Store or even App Lab!? Those are some good questions so let's address why we are on Sidequest first and not in App Lab or the Quest Store itself. To start with we are not a VR-ready app, the app found on Sidequest is actually the minimal [version] of the Android app. Why the minimal version? Well the Quest does run Android however, it's a pretty heavily modified fork with no Google services. There are no widgets, no shortcuts, and do not even allow standard notifications. As some of you may be aware Meta had only recently started to approve of what we qualify as a 2D app. We have begun creating a new version of the app specifically to meet the Quest Store requirements but it will take some time for us to get things to pass inspection. So while we are waiting we didn't want to hold back on releasing something so we can get more feedback and start getting more data for everyone to automate things.
+
+![Screenshot of App on Quest](/images/blog/2022-01-android/app_on_quest.png)
+
+Now you may be asking yourself but do I really need this installed? What am I going to gain from this? That's not easy for us to answer for every user because we all have different needs, not everyone has the same room they play in or the same devices in their smart home. By having the android app installed on your Quest you gain access to the many sensors offered by the app. Some of them are actually pretty useful. For example, our [interactive sensor] updates as soon as the screen comes on which is pretty much when you are wearing the headset. By using this sensor you could make sure your lights are on bright enough to allow room tracking to be more consistent because it does indeed require [good lighting].
```

Comment:
```suggestion
Now you may be asking yourself but do I really need this installed? What am I going to gain from this? That's not easy for us to answer for every user because we all have different needs, not everyone has the same room they play in or the same devices in their smart home.


By having the android app installed on your Quest you gain access to the many sensors offered by the app. For example, our [interactive sensor] updates as soon as the screen comes on, which means you're wearing the headset. This allows you to make sure your lights are bright enough to play VR.
```